web/packages/teleterm/src/ui/Search/pickers/ActionPicker.story.tsx

```diff
@@ -59,6 +59,7 @@ export const Items = (props: { maxWidth: string }) => {
           max-width: ${maxWidth};
           min-width: 0;
           flex: 1;
+          background-color: ${props => props.theme.colors.levels.elevated};
```

Comment:
Let's replace `<Flex gap={4}>` above with `<Flex gap={4} alignItems="flex-start">` to get rid of the second column stretching all the way down.